deploy/bin/spack-check

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash -eu
+
+CONFIG=$1
+
+. /etc/profile.d/modules.sh
+
+export MODULEPATH=""
+. ${CONFIG}
+
+RELEASE=unstable
+STAGES="libraries applications"
+VENV=$(mktemp -d)/venv
+MAKEFILE=$(mktemp)
```

Comment:
If I'm not wrong the temporary files are not deleted at the end of the script, `trap` should do the trick even in case of error with something like:
```
trap "echo Removing temporary files; rm -Rf $VENV $MAKEFILE" EXIT
```

Reply:
Done